.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
-For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**
+**Note: FOR INTERNAL USE ONLY. For support questions, please use the Education Data Architecture group in [the Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi4SAE).
```

Comment:
Just a little thing, I noticed the quip doc has this link which has extra parameters:
https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi4SAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC

The link seems to redirect to the correct page

Reply:
let's use this link. I will update the quip doc.

Reply:
thanks!